src/components/PlioQuestion.vue

```diff
@@ -99,13 +108,17 @@ import { faCheckCircle } from '@fortawesome/free-solid-svg-icons/faCheckCircle';
 import { faTimesCircle } from '@fortawesome/free-solid-svg-icons/faTimesCircle';
 library.add(faWindowClose, faCheckCircle, faTimesCircle);
 
+import SubmitButtonPointer from './tutorial/SubmitButtonPointer.vue';
+import mcqOptionsPointer from './tutorial/mcqOptionsPointer.vue';
+import CloseButtonPointer from './tutorial/CloseButtonPointer.vue';
+
 // For how long does the spinner show (in milliseconds)
 var loadTime = 1500;
 
 export default {
-  components: { LoadingSpinner },
+  components: { LoadingSpinner, SubmitButtonPointer, mcqOptionsPointer, CloseButtonPointer},
   name: "PlioQuestion",
-  props: ["plioQuestion"],
+  props: ["plioQuestion", "isTutorialComplete", "tutorialProgress"],
```

Comment:
The information if the tutorial is complete and what the progress is, that is being passed as props to `PlioQuestion.vue`